include/geolib/sensors/DepthCamera.h

```diff
@@ -236,8 +249,12 @@ class DepthCamera {
 
     inline double getOpticalTranslationY() const { return ty_; }
 
+    inline bool initialized() const { return cache_valid_; }
```

Comment:
Please add one or more docstrings to define what the behaviour of initialised() is. Does it mean the model is configured?